dashboard/test/controllers/aichat_controller_test.rb

```diff
@@ -79,59 +78,61 @@ class AichatControllerTest < ActionController::TestCase
     end
   end
 
-  #chat_completion tests
-  test 'authroized teacher has access to chat_completion test' do
+  # start_chat_completion tests
+  test 'authorized teacher has access to start_chat_completion test' do
     sign_in(@authorized_teacher1)
-    post :chat_completion, params: @valid_params_chat_completion, as: :json
+    post :start_chat_completion, params: @valid_params_chat_completion, as: :json
     assert_response :success
   end
 
-  test 'student of authorized teacher has access to chat_completion test' do
+  test 'student of authorized teacher has access to start_chat_completion test' do
     sign_in(@authorized_student1)
-    post :chat_completion, params: @valid_params_chat_completion, as: :json
+    post :start_chat_completion, params: @valid_params_chat_completion, as: :json
     assert_response :success
   end
 
-  test 'Bad request if required params are not included for chat_completion' do
+  test 'Bad request if required params are not included for start_chat_completion' do
     sign_in(@authorized_teacher1)
-    post :chat_completion, params: {newMessage: "hello"}, as: :json
+    post :start_chat_completion, params: {newMessage: "hello"}, as: :json
     assert_response :bad_request
   end
 
-  test 'Bad request if storedMessages param is not included for chat_completion' do
+  test 'Bad request if storedMessages param is not included for start_chat_completion' do
     sign_in(@authorized_teacher1)
-    post :chat_completion, params: @missing_stored_messages_params, as: :json
+    post :start_chat_completion, params: @missing_stored_messages_params, as: :json
     assert_response :bad_request
   end
 
-  test 'returns user profanity status when chat message contains profanity' do
```

Comment:
Unit tests to check that user and model profanity are detected are included in [`aichat_request_chat_completion_job_test`](https://github.com/code-dot-org/code-dot-org/blob/d712c4e1c19e60d8a66b4b2a457ec14db93d2353/dashboard/test/jobs/aichat_request_chat_completion_job_test.rb#L16-L43).